LayoutTests/fast/css/cssText-shorthand.html

```diff
@@ -1,41 +1,17 @@
 <!DOCTYPE html>
 <html>
 <body>
-<script src="../../resources/js-test-pre.js"></script>
+<script src="../../resources/js-test.js"></script>
 <script>
 
-description("This test ensures WebKit uses shorthand notations for cssText");
+description("This test ensures WebKit uses shorthand notations for cssText; most of what was originally in this test is now covered by Web Platform Tests");
 
 var tests = [
-    // FIXME: This exhibits a bug. We shouldn't be outputing border-image here.
-    ['border: 1px; border-top: 1px;', 'border: 1px;'],
-    ['border: 1px solid red;', 'border: 1px solid red;'],
-    ['border: 1px red;', 'border: 1px red;'],
-    ['border: red;', 'border: red;'],
-    ['border-top: 1px; border-right: 1px; border-bottom: 1px; border-left: 1px;', 'border: 1px;'],
-    ['border-top: 1px; border-right: 2px; border-bottom: 3px; border-left: 4px;', 'border-width: 1px 2px 3px 4px;'],
-    ['border: 1px; border-top: 2px;', 'border-width: 2px 1px 1px;'],
-    ['border: 1px; border-top: 1px !important;',
-    'border-right-width: 1px; border-bottom-width: 1px; border-left-width: 1px; border-top-width: 1px !important;'],
-
-    ['border: 1px; border-top-color: red;', 'border-width: 1px; border-top-color: red;'],
-    ['border: solid; border-style: dotted', 'border: dotted;'],
-    ['border-width: 1px;', 'border-width: 1px;'],
-
     ['-webkit-border-horizontal-spacing: 1px; -webkit-border-vertical-spacing: 2px;', 'border-spacing: 1px 2px;'],
 
-    // We don't use shorthand for font-family, etc... for compatibility reasons
+    // We don't use the font shorthand for this case yet, for historical editing compatibility reasons.
```

Comment:
We probably should at some point fwiw. The editing code will need work though.

Reply:
Yes I agree completely and would like to do this work myself if I can. I will try it after finishing with implicit.